profile.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Profile</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>
+            Looking fresh. The world needs more of your mirror selfies.
```

Comment:
These were funny. Had fun exploring the page and finding the gems

Reply:
haha thanks, helped me enjoy the challenge more